src/main/java/org/folio/dataexp/service/QuickExportService.java

```diff
@@ -0,0 +1,80 @@
+package org.folio.dataexp.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dataexp.domain.dto.ExportRequest;
+import org.folio.dataexp.domain.dto.FileDefinition;
+import org.folio.dataexp.domain.dto.QuickExportRequest;
+import org.folio.dataexp.domain.dto.QuickExportResponse;
+import org.folio.dataexp.domain.entity.ExportIdEntity;
+import org.folio.dataexp.exception.export.DataExportRequestValidationException;
+import org.folio.dataexp.repository.ExportIdEntityRepository;
+import org.folio.dataexp.repository.JobExecutionEntityRepository;
+import org.folio.dataexp.repository.JobProfileEntityRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+import static java.util.Objects.nonNull;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class QuickExportService {
+
+  private final FileDefinitionsService fileDefinitionsService;
+  private final DataExportService dataExportService;
+  private final JobProfileEntityRepository jobProfileEntityRepository;
+  private final ExportIdEntityRepository exportIdEntityRepository;
+  private final JobExecutionEntityRepository jobExecutionEntityRepository;
+
+  public QuickExportResponse postQuickExport(QuickExportRequest quickExportRequest) {
+    var fileDefinition = new FileDefinition().id(UUID.randomUUID()).size(0).fileName(quickExportRequest.getType() + "-quick.csv");
```

Comment:
Hi @obozhko-folio , according MDEXP-677 quick export pattern is quick-export-<export job id>.mrc, please rename fileName to quick-export.csv. <export job id> will be added at file uploading.

Reply:
Done.